### PR TITLE
Feature/plausible analytics

### PR DESCRIPTION
**Pull request by julia-rom**
### Description (Proposed changes, additional context, chosen approach/alternative approaches, gotchas, etc)

- added two new env vars for plausible domains
- added check if window.plausible is undefined
- sensorId and title added to trackAccordion

[review app link
](https://dtpr-sticker-feature-pl-ocpeft.herokuapp.com/)

### CODE REVIEW

**Review by tonybennett — COMMENTED** [review]
Build is failing, can review once it passes

Would guess it's because you're using template literals in the HTML file

https://create-react-app.dev/docs/adding-custom-environment-variables/#referencing-environment-variables-in-the-html

**Comment by julia-rom** [comment]
ah yeah I had a feeling that was off. I just pushed an update - feeling iffy about this syntax :

`%REACT_APP_DOMAIN_URL%,%REACT_APP_AGGREGATED_DOMAIN_URL%`

would this work like a template literal? where I can use a comma in between variables @tonybennett

**Comment by tonybennett** [comment]
Doesn't say in the docs so I'm not sure

Would suggest you experiment with it if you're not sure about & validate if it does/doesn't work, if not you'll need to search out a solution

You can always build locally and look at the build folder with `yarn build`

**Comment by julia-rom** [comment]
looks like it does - found [an example in stack overflow](https://stackoverflow.com/questions/51744375/create-react-app-using-environment-variables-in-index-html) too . It's the last comment where it shows the variable being used within a string

File: src/components/Sensors/SensorAccordion.tsx [hunk 1]
```diff
@@ -143,14 +143,17 @@ function SensorAccordion({
   const parsedInfo = markdownConverter.makeHtml(additionalInfo);
 
   const iconSrc = icon;
+  const url = window.location.pathname
+  const sensorId = url.substring(url.lastIndexOf('/') + 1);
```
Comment (tonybennett) [inline src/components/Sensors/SensorAccordion.tsx]: Does the parent component have access to this ID to pass it down? Feels fragile if we can pass it down through props

File: src/libs/analytics.ts [hunk 2]
```diff
@@ -1,49 +1,27 @@
 declare global {
   interface Window {
     fathom: any;
```
Comment (tonybennett) [inline src/libs/analytics.ts]: still using fathom? if not, can remove?

Comment (turnKei) [reply src/libs/analytics.ts]: not being used anymore

Comment (julia-rom) [reply src/libs/analytics.ts]: yep ^ updated in the plausible analytics branch that hasn't been merged in yet

File: src/components/Sensors/SensorAccordion.tsx (hunk 1 above)
Comment (julia-rom) [reply src/components/Sensors/SensorAccordion.tsx]: hmm SensorTaxonomy is the parent but doesn't have the Id, neither does SensorView.... still investigating

Comment (tonybennett) [reply src/components/Sensors/SensorAccordion.tsx]: if nothing else, it should be available as a route param, which can be passed down or re-accessed

Comment (julia-rom) [reply src/components/Sensors/SensorAccordion.tsx]: ok fixed, passed down sensorId from Sensor down to SensorAccordian (was a few components deep)

**Review by tonybennett — COMMENTED** [review]
Looks good, @turnKei validate everything is coming through on Plausible?

**Comment by turnKei** [comment]
@julia-rom Can I get the review app linked in the PR description? Helps review functionality 👍

**Comment by julia-rom** [comment]
added! seeing an application error with the heroku apps link - just investigating

**Pull request closed without merging**